src/braket_container.py

```diff
@@ -151,7 +151,13 @@ def kick_off_customer_script(entry_point: str) -> multiprocessing.Process:
         customer_module = importlib.import_module(str_module)
         customer_method = getattr(customer_module, str_method)
 
-        process_kwargs = {"target": customer_method}
+        def wrapped_customer_method(queue, **kwargs):
+            try:
+                customer_method(**kwargs)
+            except Exception as exc:
+                queue.put(exc)
```

Comment:
Do you need to re-throw the exception here so that the customer execution script will end?

Reply:
Why not just write to the output file and exit here?

Reply:
This is code that runs inside the process, but we want to "log failure and exit" from the main container process. Putting the exception in the queue lets us transfer the information from the customer method process to the main container process. Logging the failure may work properly from within the child process, but calling sys.exit won't impact the parent process like we want.

Our current codepath
- Child process start
- Child process raises an exception
- Child process joins (immediately after exception occurs)
- Check if child process exited with code 0
- If not, raise a generic exception

New codepath
- Child process starts
- Child process raises an exception
- Exception is captured and put in the queue
- Child process joins (immediately after exception occurs)
- Check if there is an exception in the queue
- If so, raise a specific exception

Reply:
> Do you need to re-throw the exception here so that the customer execution script will end?

No, since there are no more lines of code in `wrapped_customer_method`. We may want to rethrow the exception for the sake of logging; I'll check what the impact is here